clone-awx-operator.py

```diff
@@ -14,9 +14,22 @@
 import sys
 import tempfile
 
-DEFAULT_BRANCH = "devel"
 DEFAULT_AWX_OPERATOR_REPO = "https://github.com/ansible/awx-operator"
 
+# get the appVersion configured in Chart.yaml
+def get_app_version():
+    # naive implementation without yaml-parsing package
+    try:
+        with open('./.helm/starter/Chart.yaml') as chart:
+            for line in chart:
+                if line.startswith("appVersion"):
+                    result = line.replace(' ', '').split(":")
+                    if len(result) != 2:
+                        break
+                    return result[1].rstrip()
+    except:
+        raise FileNotFoundError("Failed to open Chart.yaml")
+    raise KeyError("Could not find appVersion")
```

Comment:
Maybe we can consider an alternative approach here, if you don't want to go with the yaml parser. I've been playing around with it a little. Here is what I've come up with:

```
# get the appVersion configured in Chart.yaml
def get_app_version():
    # naive implementation without yaml-parsing package
    try:
        with open("./.helm/starter/Chart.yaml") as chart:
            for line in chart:
                line = line.strip()
                if line.startswith('appVersion:'):
                    result = line.split(":", 1)
                    if len(result) != 2:
                        raise KeyError("Malformed appVersion")

                    version = result[1].rstrip()
                    if not version:
                        raise KeyError("No appVersion value found")

                    return version

        raise KeyError("Could not find appVersion")

    except FileNotFoundError:
        raise FileNotFoundError("Failed to open Chart.yaml")
```

Here I think using `strip()` might handle the whitespace better than replacing it. Plus we probably want to split on the first colon `:` only. Maybe it's a little pedantic though.

I also think it would be ideal to check for empty version values.